core/domain/question_services_test.py

```diff
@@ -87,6 +92,22 @@ def test_get_questions_by_skill_ids(self):
         self.assertEqual(
             questions[0].to_dict(), self.question.to_dict())
 
+    def test_get_questions_by_skill_ids(self):
```

Comment:
What will happen if we ask for 4 questions? Maybe add a test case for that?

Reply:
Done.